_integration/testsuite/fixtures/httpbin.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 VMware, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.  You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: &name httpbin
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: *name
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: *name
+    spec:
+      containers:
+      - name: httpbin
+        image: docker.io/kennethreitz/httpbin
```

Comment:
Should we pin to tags and not latest?

Reply:
Maybe?

Reply:
For now, I'd like to stick with latest. It also looks like neither of the 2 images we use publish anything except "latest" :)

Reply:
Ahh, I see. My feat with these types of tests is that image changes and breaks our tests and we think there's a regression. I'm always leary of using "latest" for anything.